chia/cmds/init_funcs.py

```diff
@@ -502,6 +502,7 @@ def chia_init(
             db_path_replaced = new_db_path.replace("CHALLENGE", config["selected_network"])
             db_path = path_from_root(root_path, db_path_replaced)
 
+            mkdir(db_path.parent)
```

Comment:
Nothing to do with this patch, but it weirds me out that we have a utility function with the same name as `os.mkdir`, but different behaviour.

Reply:
Agreed, that is not good